apps/agora/data/.gitignore

```diff
@@ -0,0 +1,165 @@
+.env
+
+# Downloaded data
+local/data/
```

Comment:
Specify folder used to download Synapse data. Otherwise, directly copied from `apps/openchallenges/edam-etl/.gitignore`